src/types/block/entity.rs

```diff
@@ -45,27 +62,73 @@ pub mod utils {
     pub const ITEM_SLOT_MAX: f32 = 64_f32;

```

Comment:
Add for 16, and 1

Reply:
+ other containers and the containers that match it

Mid = chest

And add them to mod (like where to find these)